matflow/data/template_components/task_schemas.yaml

```diff
@@ -995,3 +995,19 @@
       script_data_in: direct
       script_data_out: direct
       script_exe: python_script
+
+- objective: modify_VE_geometry
```

Comment:
Should this be a method of `modify_VE` objective, rather than an objective in its own right?